packages/vscode-extension/src/extension.ts

```diff
@@ -126,6 +132,15 @@ export async function activate(context: vscode.ExtensionContext) {
   // UI is ready to show & interact
   await vscode.commands.executeCommand("setContext", "fx-extension.isTeamsFx", isTeamsFxProject);
 
+  // control whether to show chat participant entries
+  await vscode.commands.executeCommand(
+    "setContext",
+    "fx-extension.isChatParticipantEnabled",
```

Comment:
Will we use this variable in `package.json`?

Reply:
Yes